bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -498,7 +498,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- Default surface roughness parameterization                         -->
 <!-- ================================================================== -->
 
-<z0param_method>ZengWang2007</z0param_method>
+<z0param_method phys="clm4_5">ZengWang2007</z0param_method>
+<z0param_method phys="clm5_0">ZengWang2007</z0param_method>
+<z0param_method phys="clm5_1">Meier2022</z0param_method>
```

Comment:
Another way to do this is to make ZengWant2007 the default, that's overridden for clm5_1. It doesn't matter much either way though. That's the way I did it in the CESM3_dev branch though.

Reply:
This is just a FYI type thing. So I'll resolve...

Reply:
Done